packages/ui/src/Integration/IntegrationDetail.tsx

```diff
@@ -0,0 +1,33 @@
+import { Nav } from 'patternfly-react';
+import * as React from 'react';
+import { Container } from '../Layout';
+
+// import { IntegrationDetailHistoryListView } from './IntegrationDetailHistoryListView';
```

Comment:
This component will eventually contain all of the tabs. Right now I only have the `IntegrationDetailTab`, which I will incorporate into here. Perhaps best to remove this component and story completely until all in-the-future-nested components are ready.

Reply:
If we handle tabs as single URLs, we shouldn't need to render all the tabs at once but just the one that we want to be rendered. And that decision should be driven by the app.

Reply:
Thanks @riccardo-forina, I think I'm understanding now and wouldn't have been able to without working on the app (e.g. just sticking to storybook). Tomorrow during our meet let's review how to best set this up with respect to the tabs within the app.